client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -105,6 +107,14 @@ public static StreamManager create(ClientConfig clientConfig) {
      */
     boolean createScope(String scopeName);
 
+    /**
+     * Gets an iterator for all streams in scope. 
+     * 
+     * @param scopeName The name of the scope for which to list streams in.
+     * @return Iterator of Stream to iterator over all streams in scope. 
+     */
+    Iterator<Stream> listStreamsInScope(String scopeName);
```

Comment:
Exercising this call in an existing integration test would be great to assess the end to end functionality.

Reply:
done. added new test called ScopeTest